Cargo.toml

```diff
@@ -69,6 +69,7 @@ encoding_rs = { version = "0.8.33", optional = true }
 profiling = "1.0.13"
 smallvec = "1.11"
 pixman = { version = "0.2.1", features = ["drm-fourcc", "sync"], optional = true }
+aliasable = "0.1.3"
```

Comment:
If this is only used by the multigpu renderer, it should probably be an optional dependency enabled by `renderer_multi`.

Reply:
done